packages/shared/src/components/badges/TopReaderBadge.tsx

```diff
@@ -0,0 +1,132 @@
+      fill="url(#goldGradient)"
+    />
+    <path
+      d="M36.001 7.536c-11.94 0-21.62 9.56-21.62 21.353 0 11.793 9.68 21.353 21.62 21.353 11.941 0 21.621-9.56 21.621-21.353 0-11.793-9.68-21.353-21.621-21.353Z"
+      fill="url(#profileImage)"
+      transform="matrix(1.01756,0,0,1.03032,-0.633698,-0.875786)"
+    />
+    <defs>
+      <linearGradient
+        id="goldGradient"
+        x1="0"
+        y1="0"
+        x2="79.5564"
+        y2="71.6044"
+        gradientUnits="userSpaceOnUse"
+      >
+        <stop stopColor={colors.bun['10']} />
+        <stop offset="0.275" stopColor={colors.cheese['10']} />
+        <stop offset="0.5" stopColor={colors.bun['10']} />
+        <stop offset="0.75" stopColor={colors.cheese['10']} />
+        <stop offset="1" stopColor={colors.bun['10']} />
+      </linearGradient>
+
+      {/* This is a hack so that we can properly center the image without fiddling with html and css */}
+      <pattern
+        id="profileImage"
+        preserveAspectRatio="xMidYMid slice"
+        x="0"
+        y="0"
+        width="1"
+        height="1"
+        viewBox="0 0 512 512"
+      >
+        <image width="512" height="512" href={imageUrl} />
+      </pattern>
+    </defs>
+  </svg>
+);
+
+export const TopReaderBadge = ({
+  topReader,
+}: {
+  topReader: {
+    id: string;
+    user?: LoggedUser;
+    issuedAt: Date;
+    keyword: Pick<Keyword, 'value' | 'flags'>;
+    image: string;
+  };
+}): ReactElement => {
+  const date = new Date(topReader?.issuedAt);
+  const issuedAt = date.toLocaleString('en-US', {
+    year: 'numeric',
+    month: 'long',
+  });
+
+  return (
+    <div
+      className="flex h-80 w-80 rounded-24 p-1 text-center text-text-primary"
+      style={{
+        backgroundImage: themeToLinearGradient[DevCardTheme.Gold],
+      }}
+    >
+      {/* @TODO: use correct image */}
+      <div className="flex max-w-full flex-1 flex-col items-center rounded-20 bg-background-default bg-[url('https://i.bratteng.com/1730314661865_xhfx89.png')] bg-cover px-3 py-5">
+        <div className="flex justify-center pb-1">
+          <BadgeIcon imageUrl={topReader?.user?.image} />
+        </div>
+
+        <TruncateText className="font-bold typo-footnote">
+          {topReader?.user?.name}
+        </TruncateText>
+
+        <TruncateText className="pb-1 text-text-secondary typo-caption1">
+          @{topReader?.user?.username}
+        </TruncateText>
+
+        <h1 className="pb-1 font-bold typo-title2">Top reader</h1>
+
+        <p className="text-text-secondary typo-caption1">{issuedAt}</p>
```

Comment:
```suggestion
        <p className="text-text-quaternary typo-caption1">{issuedAt}</p>
```
It's quarternary color in design.